README.md

```diff
@@ -52,7 +52,8 @@ var TouchID = require('react-native-touch-id');
 
 var YourComponent = React.createClass({
   _pressHandler() {
-    TouchID.authenticate()
+  var reasonText = "Your reason string for the TouchID dialogue"
```

Comment:
Let's keep the language consistent in the readme and use `Touch ID authentication` instead of `TouchID dialogue`.